lib/aio/job.py

```diff
@@ -84,6 +84,8 @@ async def run_container(job: Job, subject: Subject, ctx: JobContext, log: LogStr
 
         args = [
             *ctx.container_cmd, 'run',
+            # we run arbitrary commands in that container, which aren't prepared for being pid 1; reap zombies
+            '--init',
```

Comment:
The comment is too long, and in the wrong tone.

```python
'--init',  # make sure zombies get reaped
```

would have done it :)

Reply:
Yeah, I wanted to point out that we don't control the entry point (or at least, it's not a single one), so we can't add reapers there.